docker/README.md

```diff
@@ -2,15 +2,15 @@
 
 This directory contains the automated Docker deployment of ION.  This will create 5 docker containers that contain the core components to run ION.
 
-- bitcoin-core - This container will run the bitcoin-core client.  `NOTE: This is currenlty using testnet as the source chain`
+- bitcoin-node - This container will run the bitcoin-node client.  `NOTE: This is currently using testnet as the source chain`
```

Comment:
For my understanding, why the rename? Bitcoin Core is the official name.

Reply:
Was finding it confusing having bitcoin-core ion-core ion-bitcoin, have changed back though